### ISSUE TITLE
Make mkPoetryEnv include scripts.

### ISSUE THREAD

**Issue by takeda** [nix-community/poetry2nix#124]
This is a great project and it is almost perfect for my use case, but I'm wondering if it is possible to add a shell hook so it would install the package in editable mode, kind of how buldPythonApplication does it (https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/hooks/setuptools-build-hook.sh#L23).

What I mean is not just have packages installed and editable, but also commands defined in script section would use local source.

This example illustrates what I mean: https://github.com/takeda/example_python_project especially if you use direnv, you enter a directory and a "hello" command becomes available that belongs to the project. If you make change to the source code, the command immediately reflects that without rebuilding.

In other words I would want the behavior of `poetry shell` in the nix-shell without having to call that command.

Perhaps that's already possible and I just don't know how to set it up.

**Comment by takeda** [comment]
So I just found this demo: https://www.youtube.com/watch?v=XfqJulSAPBQ and looks like the proper way to create shell is to use `poetry2nix.mkPoetryEnv` so I suppose the actual problem is that `mkPoetryEnv` doesn't include entry points defined in `[tool.poetry.scripts]`.

**Comment by adisbladis** [comment]
Fixed in https://github.com/nix-community/poetry2nix/pull/150.
